templates/types/streaming/nextjs/app/components/ui/chat/chat-message.tsx

```diff
@@ -60,8 +61,9 @@ function ChatMessageContent({
 
   const contents: ContentDisplayConfig[] = [
     {
-      order: -4,
+      order: 1,
       component: imageData[0] ? <ChatImage data={imageData[0]} /> : null,
+      isHidden: message.role === "assistant",
```

Comment:
isn't that a backend problem that the backend is still sending this annotation for assitant's messages?

Reply:
why do we need it on the server if we don't show it on the client?